packages/2d/src/components/Video.ts

```diff
@@ -263,11 +281,30 @@ export class Video extends Rect {
     }
   }
 
+  protected setPlaybackRate(playbackRate: number) {
```

Comment:
For the sake of completeness, the value passed here is not a number but a `SignalValue<number>`
As mentioned previously, this implementation won't work with functions so we should check for that here and log a warning:
```suggestion
  protected setPlaybackRate(playbackRate: SignalValue<number>) {
    let value: number;
    if (isReactive(playbackRate)) {
      value = playbackRate();
      useLogger().warn({
        message: 'Invalid value set as the playback rate',
        remarks: ...,
        inspect: this.key,
      });
    } else {
      value = playbackRate;
    }
```